Sources/SwifterSwift/StoreKit/SKProductExtensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SKProductExtensions.swift
+//  SwifterSwift
+//
+//  Created by Batuhan Saka on 26.12.2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+#if canImport(StoreKit)
+import StoreKit
+
+public extension SKProduct {
+
+    /// SwifterSwift: Localized price of SKProduct
+    var localizedPrice: String? {
+        let formatter = NumberFormatter()
```

Comment:
Formatters are notoriously heavy performance-wise. If somebody is using this on an array of products you'll get a huge performance hit. What might be a good idea here is to create a reusable price formatter:
```Swift
private static let priceFormatter: NumberFormatter = {
    let priceFormatter = NumberFormatter()
    priceFormatter.numberStyle = .currency
    return priceFormatter
}()
```
You'll still need to update the locale in your function, but you can save recreating the formatter. The only problem is that it breaks our guideline of each extension being self-contained. What do you think @LucianoPAlmeida @omaralbeik and others?

Reply:
Of course. I will take care of that. Your suggestion will be better in case over use scenario.

Reply:
Performance improvement has been applied. @guykogus

Reply:
@guykogus To me this is fine... I think this rule could be discussed if we really have a good reason, and in this case, seems like a good one :) @omaralbeik what you think?